content/posts/shortcodes/index.fr.md

```diff
@@ -1,73 +1,152 @@
 ---
-title: "Codes abrégés"
+
+title: "Les Shortcodes"
 date: 2023-11-05T14:06:25+02:00
 description: "Codes abrégés"
 menu:
   sidebar:
-    name: "Codes abrégés"
+    name: "Les Shortcodes"
     identifier: shortcodes
     weight: 700
 hero: boat.jpg
 ---
-Ce billet échantillon est destiné à tester les éléments suivants :
+
+Ce billet d'échantillon est destiné à tester les éléments suivants :
 
 - Manipulation d'une image.
 - Différents shortcodes.
 
 ## Alerte
 
-Les alertes suivantes sont disponibles dans ce thème.
+Ce thème propose différents types d'alertes pour votre publication. Par exemple, si on ajoute le code ci-dessous :
+
+```markdown
+{{</* alert type="success" */>}}
+This is sample alert with `type="success"`.
+{{</* /alert */>}}
+```
+
+L'alerte qui apparaîtra ressemblera à ça:
 
 {{< alert type="success" >}}
-Voici une alerte avec `type="success"`.
+This is sample alert with `type="success"`.
 {{< /alert >}}
 
+Les alertes suivantes sont également disponible dans ce thème.
+```markdown
+{{</* alert type="danger" */>}}
+This is sample alert with `type="danger"`.
+{{</* /alert */>}}
+```
+
 {{< alert type="danger" >}}
-Voici une alerte avec `type="danger"`.
+This is sample alert with `type="danger"`.
 {{< /alert >}}
 
+```markdown
+{{</* alert type="warning" */>}}
+This is sample alert with `type="warning"`.
+{{</* /alert */>}}
+```
+
 {{< alert type="warning" >}}
-Voici une alerte avec `type="warning"`.
+This is sample alert with `type="warning"`.
 {{< /alert >}}
 
+```markdown
+{{< alert type="info" */>}}
```

Comment:
You are missing the beginning `/*` here.

Reply:
fixed